authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -361,9 +361,9 @@ private void removeTableMetadataObject(AuthorizationMetadataObject authzMetadata
         authzMetadataObject instanceof PathBasedMetadataObject,
         "The metadata object must be a PathBasedMetadataObject");
     Preconditions.checkArgument(
-        authzMetadataObject.names().size() == 3, "The metadata object names must be 3");
+        authzMetadataObject.names().size() == 3, "The metadata object size must be 3");
```

Comment:
Em ... What is "metadata object size"?

Reply:
"Names must be 3" makes no sense, "size must be 3" does. This message is aimed at programmers, and they would understand that it needs to have 3 elements in it.

Reply:
OK.
Then how do I interpret "authzMetadataObject.names().size()" in plain English?

Reply:
The size of the name or the metadata's object size or the metadata's name size, or if you really want to be verbose, the number of elements in the metadata's object's name.

Reply:
Metadata object has a name identifier. The name identifier consists of names. This should be size of names instead of size of object.

Reply:
@justinmclean I think this name may not correct enough.

Reply:
Note that the changes above are outdated, so you'll not see them above. I've changed them to all be consistent with each other.